functional_tests/features/bundle_ui.feature

```diff
@@ -0,0 +1,133 @@
+Feature: UI Bundle Happy Paths
+    """
+    User Role Bundle Life Cycle Happy Path
+                                 | Search | Create | Edit | Preview | Approve |
+        Publishing Admin         | ToDo   | Can    | Can  | Can     | N/A     |
+            As Creator of Bundle | N/A    | N/A    | N/A  | N/A     | Cannot  |
```

Comment:
To clarify something, currently the user who created the bundle _can_ approve the bundle themselves, it is pages that require a different user approval.